### PR TITLE
kubevirt: Add ci scripts

### PR DESCRIPTION
**Pull request by yaacov**
Add two ci scripts for installing virtctl and for storage setup

### CODE REVIEW

**Comment by yaacov** [comment]
@glekner @pcbailey please revew

**Comment by pcbailey** [comment]
@yaacov Are you going to replace pieces of [deploy-kubevirt-gating.sh](https://github.com/openshift/console/blob/master/frontend/packages/kubevirt-plugin/integration-tests/deploy-kubevirt-gating.sh) with these?

**Comment by yaacov** [comment]
> @yaacov Are you going to replace pieces of deploy-kubevirt-gating.sh with these?

yes

File: frontend/packages/kubevirt-plugin/integration-tests/ci-scripts/setup-storage.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
```
Comment (pcbailey) [inline frontend/packages/kubevirt-plugin/integration-tests/ci-scripts/setup-storage.sh:1]: Extra line

Comment (yaacov) [reply frontend/packages/kubevirt-plugin/integration-tests/ci-scripts/setup-storage.sh]: thanks +1, fixed

**Pull request merged**